core/jazz_test-lambda/components/utils.js

```diff
@@ -34,6 +34,16 @@ var validateARN = (arn) => {
   }
 
 };
+
+var validateEndpoint = (arn) => {
```

Comment:
Prefer `endpoint` instead of `arn` as the input argument to the function :)

Reply:
fixed